src/components/pill.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { Box, css, Image, get } from 'theme-ui';
+
+import { InlineFlex, Flex } from './index';
```

Comment:
- [ ] We should not import components like this from 'index'., bcoz, you are exporting this component from 'index'. There is a high chance that this might yield to `Circular Dependency` error.

You should import `InlineFlex` and `Flex` component from 'layout' instead of importing it from 'index'.